rin_serin/week04/mission02/src/components/Card.jsx

```diff
@@ -0,0 +1,62 @@
+import styled from "styled-components";
+import { useState } from "react";
+import { useNavigate } from "react-router-dom";
+import { Navigate } from "react-router-dom";
+import { Link, replace } from "react-router-dom";
+
+const CardDiv = styled.button`
+    margin: 10px;
+    width: 160px;
+    padding: 10px;
+    height: fit-content;
+    background: none;
+    border: none;
+    display: flex;
+    flex-direction: column;
+    box-sizing: border-box;
+`
+const MovieImg = styled.img`
+    width: 160px;
+    height: 240px;
+    border-radius: 12px;
+    transition: opacity 0.3s ease-in-out;
+    filter: ${props =>(props.hoverOn ? 'brightness(50%)' : 'brightness(100%)')};
```

Comment:
```suggestion
    brightness(100%);
    
    &:hover {
      brightness(50%);
    }
```

css에 내장된 hover 이용하시면 prop으로 내리지 않아도 간단하게 구현가능합니다용